package.json

```diff
@@ -13,7 +13,8 @@
         "i18next": "19.4.5",
         "immutable": "4.0.0-rc.12",
         "lodash": "4.17.19",
-        "query-string": "6.13.1"
+        "query-string": "6.13.1",
+        "@types/lodash": "4.14.108"
```

Comment:
Nitpick: As in the devDeps section, `@types` would come first in list of deps

Reply:
Fixed.